c3po/api/__init__.py

```diff
@@ -12,4 +13,5 @@
           description='a boilerplate for flask restplus web service'
           )
 
-api.add_namespace(test_ns, path='/api')
\ No newline at end of file
+api.add_namespace(test_ns, path='/api')
+api.add_namespace(feed_ns, path='/feed')
```

Comment:
This should be /api/feed right? Since we're prefixing all namespaces with /api

Reply:
Yup, gotcha. Will fix in next commit